### DISCUSSION TITLE
error in SVG_URI : lesson 11, Mood NFT

### DISCUSSION THREAD

**Discussion by 0xtdey** [Cyfrin/foundry-full-course-cu#306] [Q&A] [answered]
<img width="691" alt="image" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/91402167/c2a79984-04c0-4f22-ad3f-d4119f9f45c0">

I am not able to understand why it shows a error when i am just copying the base64 outcome.

**Comment by Naman1729** [comment, marked as answer]
Hey @tamoghna-dey , paste you SVG_URI in single line do not use multiple line .

**Reply by 0xtdey** [reply]
Yaa got it, solved the issue. Thanks
